source/fe/fe_trace.cc

```diff
@@ -49,6 +49,13 @@ FE_TraceQ<dim,spacedim>::FE_TraceQ (const unsigned int degree)
 
   // Initialize face support points
   this->unit_face_support_points = fe_q.get_unit_face_support_points();
+
+  // initialize unit support points (this makes it possible to assign initial
+  // values to FE_TraceQ)
+  this->unit_support_points.resize(this->dofs_per_cell);
+  for (unsigned int i=0; i<this->dofs_per_cell; ++i)
+    this->unit_support_points[i] = fe_q.get_unit_support_points()[i];
```

Comment:
This means that FE_TraceQ has the same unit_support_points as FE_Q, doesn't it?
Is this really what we want? Should it not just be the support_points on the lower-dimensional objects?